crates/gitbutler-tauri/desktopTemplate.hbs

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
This extra desktop entry template file (and the line pointing to it in the various `tauri.json`'s are no longer necessary. 

This was only used to force it to use x11, as opposed to wayland, because there was a bug with gnome+wayland in which it was impossible to resize the window and click+drag the window on its title bar. However this underlying bug seems to be fixed in the latest `tao` version, released as of ~2 days ago.